src/OrbitGl/GraphTrack.cpp

```diff
@@ -71,12 +66,18 @@ void GraphTrack<Dimension>::Draw(Batcher& batcher, TextRenderer& text_renderer,
   float point_x = time_graph_->GetWorldFromTick(label_time);
   float point_y = GetLabelYFromValues(values);
   std::string text = GetLabelTextFromValues(values);
-  DrawLabel(batcher, text_renderer, Vec2(point_x, point_y), text, kBlack, kWhite, label_z);
+  const Color kLabelTextColor(0, 0, 0, 255);
+  const Color kLabelBackgroundColor(255, 255, 255, 100);
```

Comment:
Could I ask why we change the name of the colors? I rather would prefer to have kTransparentWhite and kBlack

Reply:
Changed back to use kBlack and kTransparentWhite. Thanks!